lib/blocks.js

```diff
@@ -80,17 +127,83 @@ BlockController.prototype.show = function(req, res) {
   }
 };
 
-BlockController.prototype.blockIndex = function(req, res, next, height) {
-  var info = this.node.services.bitcoind.getBlockIndex(parseInt(height));
-  if(!info) {
-    return common.handleErrors(null, res);
+BlockController.prototype.showRaw = function(req, res) {
+  if (req.rawBlock) {
+    res.jsonp(req.rawBlock);
   }
+};
 
-  res.jsonp({
-    blockHash: info.hash
+BlockController.prototype.blockIndex = function(req, res, next, height) {
+  this.node.services.bitcoind.getBlockHeader(parseInt(height), function(err, info) {
+    if (err) {
+      return common.handleErrors(err, res);
+    }
+    res.jsonp({
+      blockHash: info.hash
+    });
   });
 };
 
+BlockController.prototype._getBlockSummary = function(hash, moreTs, next) {
```

Comment:
Can we use a different name than "moreTs"? I don't understand what that is supposed to mean.